### ISSUE TITLE
Running lint script with outputPath/outFile parameters throws schema validation error

### ISSUE THREAD

**Issue by lodybo** [angular-eslint/angular-eslint#535]
**Description and reproduction of the issue**
I'm trying to configure `@angular-eslint` to output a formatted JSON report into our `coverage/` directory. I added the following configuration to `angular.json`:

![lint-config](https://user-images.githubusercontent.com/1598330/121683468-ba68eb80-cabd-11eb-90a5-0c3743929896.png)

When running `npm run lint`, we get the following error in the terminal:

![lint-error](https://user-images.githubusercontent.com/1598330/121683598-e5533f80-cabd-11eb-9b8d-d8b7a0da8c93.png)

I reconfirmed the issue in a reduced test case: https://github.com/lodybo/angular-eslint-output-issue
Just clone it, install everything and run `npm run lint`.

**Versions**
(I ran these commands in my reduced test case repo.)

| package                            | version |
| ---------------------------------- | ------- |
| `@angular-eslint/builder`          | `12.1.0` |
| `ESLint`                           | `7.27.0` |
| `node`                             | `14.17.0` |

```sh
# Please run `npx ng version` in your project and paste the full output here:
Angular CLI: 12.0.2
Node: 14.17.0
Package Manager: npm 6.14.13
OS: linux x64

Angular: 12.0.2
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1200.2
@angular-devkit/build-angular   12.0.2
@angular-devkit/core            12.0.2
@angular-devkit/schematics      12.0.2
@schematics/angular             12.0.2
rxjs                            6.6.7
typescript                      4.2.4
```

- [x] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here. => I updated Angular CLI and created a new Angular app + added @angular-eslint right before submitting this ticket.

**Comment by JamesHenry** [comment]
Duplicate of https://github.com/angular-eslint/angular-eslint/issues/349, this is a feature request.

please feel free to open a PR!

**Comment by JamesHenry** [comment]
Until the feature is implemented you can always pipe the output to a file using your shell

**Comment by lodybo** [comment]
I will try that, thank you. And I'll take a look at #349, maybe I can figure out a way to implement that.

Thanks!

**Comment by JamesHenry** [comment]
That would be highly appreciated @lodybo thank you. If you going to take this on the I have shortcut for you:

We already implemented outputFile in the Nx version of the builder: https://github.com/nrwl/nx/blob/c7a5f6d7dd04f85386465dd0b7d86bcc74daa4ba/packages/linter/src/executors/eslint/lint.impl.ts#L108

You can use this (and its associated tests) as inspiration
